lib/MusicBrainz/Script/MBDump.pm

```diff
@@ -172,13 +172,13 @@ sub make_tar {
     system
         'bash', '-c',
             'set -o pipefail; ' .
-            'tar' .
+            'ionice -c3 tar' .
```

Comment:
I think it would be safer to use `--ignore` `ionice` option to let the script run even if it fails to set the requested priority.
This capability may be disabled (user, docker, kernel, or whatever).
Also a check for `ionice` command existence is likely safer too.

Basically we should make `ionice` optional, if it is there and is working, we use it, else just run stuff without it.

Also I would recommend full option names for readability.

Something like:

```bash
ioniceit() { if type ionice >/dev/null 2>&1; then ionice --class 3 --ignore $@ ; else $@; fi }

ioniceit command_to_ionice
```

Reply:
Added (with quoted "$@") in commit <https://github.com/metabrainz/musicbrainz-server/pull/2350/commits/96f56fbfac57290527537d71eaaa02a8a3cbf655>.